examples/gnrc_border_router/README.md

```diff
@@ -8,6 +8,35 @@ and routes on the BR.
 
 The script `start_network.sh` enables a *ready-to-use* BR in only one command.
 
+## Uplink
+
+The border router will route packets between a 6Lo network (PAN) and a 'normal'
+IPv6 network (i.e. the Internet).
+
+This requires the border router to have two interfaces, a downstream, 6lo-interface
+and an IPv6 uplink.
+
+This setup comes with support for three uplink types pre-configured.
+To select e.g. a SLIP uplink, add `UPLINK=slip` to your `make` command.
+
+`ethos` and `slip` will make use of the existing serial interface that is used for the
+RIOT shell to provide an upstream interface. Your computer will act as the upstream
+router, stdio is multiplexed over the same line.
+
+The `wifi` uplink will connect to an existing WiFi (IEEE 802.11) network.
+The network must provide a DHCPv6 server that supports prefix delegation (IA_PD).
+
+Use `WIFI_SSID="SSID" WIFI_PASS="password"` in your `make` command to set your WiFi
+credentials. You can alternatively edit the `Makefile`.
+
+Currently this requires an esp8266 or esp32 for the border router.
+On the downstream interface the proprietary `esp_now` link is used. It behaves similar
+to IEEE 802.15.4 but is only availiably on Espressif's wireless MCUs.
```

Comment:
Is the info about `esp_now` important? It seems more of a side note (as one can also use an external IEEE 802.15.4 radio as noted below). I rather would remove this as this is not related to the WiFi `UPLINK` configuration but rather how the user configures their downlink.

Reply:
Well `esp_now` is always available without the need to connect anything, so I think it's worth mentioning.
I've reworded it a bit though.